docs/source-2.0/spec/http-bindings.rst

```diff
@@ -550,12 +552,12 @@ Conflicts with
   :rfc:`7231#section-7.1.1.1`. The :ref:`timestampFormat-trait` MAY be used
   to use a custom serialization format.
 
-.. rubric:: Do not put too much data in HTTP headers
+.. note::
```

Comment:
Should this also be `::important`?

Reply:
I think it's more consistent with other `note`s in the spec than other `important` admonitions.